Mathlib/Topology/Order/LowerUpperTopology.lean

```diff
@@ -264,7 +444,24 @@ instance [Preorder α] [TopologicalSpace α] [LowerTopology α] [OrderBot α] [P
     · exact GenerateOpen.basic _ ⟨(a, ⊥), by simp [Ici_prod_eq, prod_univ]⟩
     · exact GenerateOpen.basic _ ⟨(⊥, b), by simp [Ici_prod_eq, univ_prod]⟩
 
-section CompleteLattice
+instance [Preorder α] [TopologicalSpace α] [UpperTopology α] [OrderTop α] [Preorder β]
+    [TopologicalSpace β] [UpperTopology β] [OrderTop β] : UpperTopology (α × β) where
```

Comment:
```suggestion
instance instUpperTopologyProd [Preorder α] [TopologicalSpace α] [UpperTopology α] [OrderTop α]
    [Preorder β] [TopologicalSpace β] [UpperTopology β] [OrderTop β] : UpperTopology (α × β) where
```
The auto-generated name is way too long. Could you also fix the lower version similarly?